package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-filings-ui",
-  "version": "7.0.40",
```

Comment:
Please rebase so that this shows 7.0.42, and then maybe re-test to ensure that the other changes don't affect your updates.

Reply:
OK, you're rebased. Now please update the app version again 😛